src/SFA.DAS.PensionsRegulatorApi.UnitTests/Controllers/Given_Handler_Throws_Exception.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using NUnit.Framework;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Controllers;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Controllers
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_Handler_Throws_Exception
```

Comment:
As per comment on other test.. remove `Handler` from test name; it will be `IMediator` we will be mocking.

Reply:
sure will sort